### DISCUSSION TITLE
2024-06-05(수)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#102] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일

### 해야할 일
- 토큰 기반 인증 처리

### 어려운 일
- 일단 쿠키에 토큰 저장하는 것부터 어렵군요ㅠ

**Comment by q45402sk** [comment]
### 완료한 일
zzim 컴포넌트

### 해야할 일
drawer 컴포넌트

### 어려운 일

**Comment by devwqc** [comment]
### 완료한 일
- BackButton, 회원가입 후 route 기능에 대한 고민

### 해야할 일
- 헤더 + 레이아웃 최종 작업
- 홈 페이지 및 상품 목록 리스트

### 어려운 일
- 파이팅!💪

**Comment by minyoung0503** [comment]
### 완료한 일
리뷰 전체페이지 layout

### 해야할 일
리뷰 정렬
공동구매 layout

### 어려운 일
